security/auto-host-endpoints.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Protect Kubernetes nodes
+description: Protect Kubernetes nodes with host endpoints managed by {{site.prodname}}
+---
+
+### Big picture
+
+Secure Kubernetes nodes with host endpoints managed by {{site.prodname}}.
+
+### Value
+
+{{site.prodname}} can automatically create host endpoints for your Kubernetes nodes. The lifecycle of these host endpoints are managed by {{site.prodname}} in response to an evolving cluster to ensure policy protecting your nodes is always enforced.
+
+### Features
+
+This how-to guide uses the following Calico features:
+- **HostEndpoint**
+- **Profile**
+- **KubeControllersConfiguration**
+- **GlobalNetworkPolicy**
+
+### Concepts
+
+### Host endpoints
+
+Each host has one or more network interfaces that it uses to communicate externally. You can use {{site.prodname}} network policy to secure these interfaces (called host endpoints).
+{{site.prodname}} host endpoints can have labels, and they work the same as labels on workload endpoints. The network policy rules can apply to both workload and host endpoints using label selectors.
+
+Host endpoints come in two types: *named* and *wildcard*. Named host endpoints secure a specific interface such as "eth0", and are created by setting `interfaceName: <name-of-that-interface>` -- for example, `interfaceName: eth0`.
+Wildcard host endpoints secure _all_ of the hosts interfaces non-workload interfaces. They are created by setting `interfaceName: "*"`.
```

Comment:
I hope I'm not getting confused or misremembering, but I thought wildcarded represents all interfaces, including workload interfaces, and that you could use policy on them to protect the host from workloads?  (I know there is the host to workload exception for health checks though.)

Reply:
No you're right it does but in the context of apply-on-forward policies.